overlays/kubernetes/canonical-k8s-base.yaml

```diff
@@ -0,0 +1,6 @@
+# Variables
+k8s_channel:                     &k8s_channel                    __K8S_CHANNEL__
+
+relations:
+- [ 'k8s-worker:cluster', 'k8s:k8s-cluster' ]
+- [ 'k8s:containerd', 'k8s-worker:containerd' ]
```

Comment:
We should add
```yaml
- [ 'k8s:cos-worker-tokens', 'k8s-worker:cos-tokens' ]
```
to this file to pass authentication tokens to the workers. These are needed for grafana-agent for example.